FlyveMDMInventoryTests/BatteryTest.swift

```diff
@@ -23,6 +23,7 @@ class BatteryTest: XCTestCase {
         super.tearDown()
     }
 
+    /// Get the state of the battery
```

Comment:
3 slashes?

Reply:
@ajsb85 in swift you may use /** ... */ for multi-line or /// for single-line comments

Reply:
Yes, Hector explained to me that it is with 3 slashes the single line documentation so Jazzy can read the comment properly